.gitignore

```diff
@@ -16,3 +16,5 @@ api/media
 api/webapp/build
 api/scripts/data
 secret_keys.json
+
+custom\.yml
```

Comment:
Weird that there's a `\` character here. Remove it next time you get the chance.